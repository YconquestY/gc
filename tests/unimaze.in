20
8
2018
