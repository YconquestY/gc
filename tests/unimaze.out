 Maze width: Maze height: Random seed: ┌────┬┬──┬┬─┬─┬─┬─┐
│╷╷╷╷╵│╷╶┤│╷╵╶┼┐└╴│
│├┤└┤╷└┤╷╵││┌─┘└─╴│
├┘└╴└┴┬┴┘╷╵└┘┌╴┌╴╷│
│╷┌╴╶┐└┬╴├┬─╴├─┤┌┘│
│└┤╷┌┘╷╵╷│╵╶┐├╴└┴╴│
└─┴┴┴─┴─┴┴──┴┴────┘
